### PR TITLE
Query SQL databases

### LINKED ISSUE #1738: More database integrations on sql.from() page
There are some relational databases that _I believe_ (please confirm) we can pull data from using `sql.from()`, which aren't listed on the [sql.from() doc page](https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/sql/from/#query-an-sqlite-database):

- MariaDB, using a [MySQL client](https://stackoverflow.com/questions/57156119/does-mysqls-connector-j-work-with-mariadb)
- CockroachDB, using a [Postgres driver](https://www.cockroachlabs.com/blog/why-postgres/)
- AWS RDS, using a MySQL, Postgres, or Microsoft SQL Server driver
- Google Bigtable, using an [experimental driver](https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/experimental/bigtable/from/)
- SAP Hana (perhaps not quite landed?)

If these do indeed work for pulling in relational data, can we please include them on this page? 

Also, can we please link to the [BigTable page](https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/experimental/bigtable/from/) on this page as well, to make it easier to find?

Reason, for technical differentiation, we want to show how InfluxDB provides more alternatives for time series data enrichment than alternatives. It'd be very helpful if this page could provide the proof points to back up a slide like the one below:

image

##### Relevant URLs
- https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/sql/from/
- https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/experimental/bigtable/from/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1738

Adds pages for each unique SQL database supported by Flux

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/flux/v0.x/query-data/sql/percona.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+##### On this page
+
+- [Data source name](#data-source-name)
+- [Data types](#data-types)
+- [Results structure](#results-structure)
+- [Store sensitive credentials as secrets](#store-sensitive-credentials-as-secrets)
+
+## Data source name
+The `mysql` driver uses the following data source name (DSN) syntax:
```
Comment (pierwill) [inline content/flux/v0.x/query-data/sql/percona.md:52]: Percona?

File: content/flux/v0.x/query-data/sql/sqlite.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+To query SQLite, Flux must have access to the filesystem.
+If Flux does not have access to the file system, the query will return an error
+similar to one of the following:
+
+- `Error: unable to open database file`
+- `failed to read file: filesystem service is uninitialized`
+- `An internal error has occurred`
+
+If using InfluxDB Cloud or InfluxDB OSS, the Flux process **does not** have 
+access to the filesystem.
```
Comment (pierwill) [inline content/flux/v0.x/query-data/sql/sqlite.md:56]: Maybe a note about how you must use the REPL to do this?

**Review by pierwill — APPROVED** [review]
👏

File: content/flux/v0.x/query-data/sql/percona.md (hunk 1 above)
Comment (sanderson) [reply content/flux/v0.x/query-data/sql/percona.md:52]: Percona is just a fork of MySQL and uses the same driver.

File: content/flux/v0.x/query-data/sql/sqlite.md (hunk 2 above)
Comment (sanderson) [reply content/flux/v0.x/query-data/sql/sqlite.md:56]: The REPL isn't the only other way to run Flux. Users can build their own implementations, so I hesitate to call the REPL out specifically here.

**Review by kelseiv — COMMENTED** [review]
For procedures, thoughts on using a step-by-step pattern:

1. Store your SQL database credentials as an InfluxDB secret. See how to add a secret.
2. Use secrets() in your query to retrieve SQL credentials.
3. Import the sql package.
4. Use sql.from(), including the following parameters:
   - your SQL driverName
   - dataSourceName
   - query to execute

Then maybe include an input/output example to show results structure?

We could roll the following sections into one conceptual/overview doc with tabbed content for each supported SQL db:

Overview of SQL databases
	- Data source name
	- Data types
	- Results structure

**Comment by sanderson** [comment]
@kelseiv 

With the restructure of the Flux documentation, we're change the context that Flux lives in and treating it more as a standalone product rather than being tied directly to InfluxDB. With that context:

> 1. Store your SQL database credentials as an InfluxDB secret. See how to add a secret.
> 2. Use secrets() in your query to retrieve SQL credentials.

This only applies if they're using InfluxDB. The `secrets` package integrates with the InfluxDB secrets API. That's why this is separated out.

> 3. Import the sql package.
> 4. Use sql.from(), including the following parameters:
>    - your SQL driverName
>    - dataSourceName
>    - query to execute

I'll structure these steps as a procedural ordered list.

> We could roll the following sections into one conceptual/overview doc with tabbed content for each supported SQL db:
>
> Overview of SQL databases
>    - Data source name
>    - Data types
>    - Results structure

The data source name (DSN) syntax and data conversions are unique to each driver. To pull these into a tabbed overview doc means that users viewing the   "Query PostgreSQL" doc would have to leave the page to get the DSN/data type conversions.

The results structure can be pulled into the overview doc since it's the same for all of them. I'll just include a really generic example output table since the output wholly depends on the schema of the data in the queried database.

**Pull request merged**